imagely.bin.js

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
-var imagely = require('../imagely');
+require('babel-core/register');
```

Comment:
Looks like the recommendation is to call `require('babel-register')` directly
http://stackoverflow.com/a/37780995/1229866